src/AutoComplete/AutoComplete.js

```diff
@@ -497,7 +497,7 @@ class AutoComplete extends Component {
         />
         <Popover
           {...popoverProps}
-          style={styles.popover}
+          style={prepareStyles(Object.assign(styles.popover, (popoverProps || {}).style))}
```

Comment:
Thank you ! It doesn't throw if we try destructuring popoverProps that are undefined ?